esp-hal/src/uart.rs

```diff
@@ -2432,8 +2407,8 @@ impl Info {
     ///
     /// # Errors
     ///
-    /// [`Err(ConfigError::UnsupportedFifoThreshold)`][ConfigError::UnsupportedFifoThreshold] if provided value exceeds maximum value
-    /// for SOC :
+    /// [ConfigError::UnsupportedFifoThreshold] if provided value exceeds
```

Comment:
is it useful to repeat the description from the enum (but this time with values in hex) - same for set_rx_timeout

Reply:
mhhhh - then the duplication is even more useless 😄

Reply:
Will be removed in https://github.com/esp-rs/esp-hal/pull/3142/commits/74c088fce28ee4335f157394d866d4102a83d484